modules/openapi-generator/src/main/resources/JavaSpring/libraries/spring-boot/requestMapping.mustache

```diff
@@ -0,0 +1,3 @@
+{{=<% %>=}}
```

Comment:
It's the same one for spring-boot and spring-mvc, maybe it can be the default value of the spring generator and override only for spring-cloud?